src/server/MapPlaylist.ts

```diff
@@ -41,6 +41,7 @@ const frequency: Partial<Record<GameMapName, number>> = {
   Mars: 3,
   Mena: 6,
   Montreal: 6,
+  MiddleEast: 5,
```

Comment:
```suggestion
  MiddleEast: 6,
```
New maps normally get higher frequencies.